### PR TITLE
Remove incorrect 3D suffix from nodes in realtime navigation

### LINKED ISSUE #6711: Stable "Real Time Navigation" uses node names with "3D" suffix from 4.x
**Your Godot version:**
Godot 3.5.1

**Issue description:**
Docs mention `NavigationAgent3D` and `NavigationObstacle3D` which are the new names in Godot 4. The "3D" suffix should be dropped.

**URL to the documentation page:**
https://docs.godotengine.org/en/stable/tutorials/navigation/real_time_navigation_3d.html

### PR DESCRIPTION
**Pull request by winston-yallow**
Fixes #6711

### CODE REVIEW

File: tutorials/navigation/real_time_navigation_3d.rst [hunk 1]
```diff
@@ -83,7 +83,7 @@ agents, this is the :ref:`max neighbors<class_NavigationAgent_property_max_neigh
 property of an agent and can be adjusted. This is **not** a limit for
 how many agents can use a navigation region at the same time.
 
-NavigationObstacle3D
+NavigationObstacle
 --------------------
```
Comment (skyace65) [inline tutorials/navigation/real_time_navigation_3d.rst:87]:
```suggestion
NavigationObstacle
------------------
```

File: tutorials/navigation/real_time_navigation_3d.rst [hunk 2]
```diff
@@ -55,7 +55,7 @@ to each other. Additionally a baked navmesh can be moved at runtime and
 agents will still be able to navigate onto it from another region.
 For example, navigating onto a moving platform that has stopped will work.
 
-NavigationAgent3D
+NavigationAgent
 -----------------
```
Comment (skyace65) [inline tutorials/navigation/real_time_navigation_3d.rst:59]:
```suggestion
NavigationAgent
---------------
```

**Review by skyace65 — CHANGES_REQUESTED** [review]
The lines underneath need to be changed as well.

**Review by YuriSizov — APPROVED** [review]

**Comment by YuriSizov** [comment]
Thanks!

**Pull request merged**